docs/user/installation.rst

```diff
@@ -121,6 +133,53 @@ PyPi
 
 `intelmqsetup` will create all necessary directories, provides a default configuration for new setups. See the :ref:`configuration` for more information on them and how to influence them.
 
+Docker **with** docker-compose (recommended)
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Navigate to your preferred installation directory, i. e. use ``mkdir ~/intelmq && cd ~/intelmq``
+
+.. code-block:: bash
+
+   git clone https://github.com/certat/intelmq-docker.git
+
+   sudo docker pull certat/intelmq-full:1.0
+
+   mkdir intelmq_logs
+
+   cd intelmq-docker
+
+   chown -R $USER:$USER example_config
```

Comment:
This is only needed if the directory `example_config` does not exist. In that case docker-compose (which is run as root) creates the directory with root permissions. Given that by default the permissions should be oke, but apparently there can occur permission problems in some situations, I would add a sentence below: "Note: If you get a `Permission denied` error from one of the containers, please change the permissions back using `chown -R $USER:$USER example_config`"